scripts/generate-owner-certs.sh

```diff
@@ -2,14 +2,15 @@
 
 source "$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/cert-utils.sh"
 
-ENV_FILE="/etc/sysconfig/go-fdo-server-owner"
-[ ! -f "${ENV_FILE}" ] || source "${ENV_FILE}"
-
-conf_dir="${OWNER_CONF_DIR:-/etc/go-fdo-server}"
-subj="${OWNER_SUBJECT:-/C=US/O=FDO/CN=Owner}"
-key="${OWNER_KEY:-${conf_dir}/owner.key}"
-crt="${OWNER_CRT:-${conf_dir}/owner.crt}"
+cert_dir="/etc/pki/go-fdo-server"
+subj="/C=US/O=FDO/CN=Owner"
+key="${cert_dir}/owner-example.key"
+crt="${cert_dir}/owner-example.crt"
 
+# Do not overwrite existing cert/key files unless one of the pair is
+# missing
+if [[ ! -f "${key}" || ! -f "${crt}" ]]; then
+  rm -f "${key}" "${crt}"
+fi
 generate_cert "${key}" "${crt}" "${subj}"
-
-"$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/generate-device-ca-certs.sh"
+chmod g+r "${key}"
```

Comment:
The owner private key must belong and it must be readable by the owner user only as it's the only one using it for signing.

The onwer cert must belong to the owner user and the group permissions are not important because it's not used by any other service (it's used in tests only by the root user to extract the public key for the resale protocol)